src/Task/Remote/Ssh.php

```diff
@@ -117,6 +134,12 @@ public function getCommand()
         foreach ($this->exec as $command) {
             $commands[] = $this->receiveCommand($command);
         }
+
+        $remoteDir = $this->remoteDir ? $this->remoteDir : $this->getConfigValue('remoteDir');
+        if (!empty($remoteDir)) {
+            $this->printTaskInfo(sprintf('Working directory "%s"', $remoteDir));
```

Comment:
`getCommand` is not supposed to print something to output, as it can be used as parameter to other task and not executed directly.